corehq/apps/export/tasks.py

```diff
@@ -112,16 +112,10 @@ def rebuild_saved_export(export_instance_id, manual=False):
     """
     download_data = _get_saved_export_download_data(export_instance_id)
     status = get_task_status(download_data.task)
-    if manual:
-        if status.not_started() or status.missing():
-            # cancel pending task before kicking off a new one
-            if download_data.task:
-                download_data.task.revoke()
-        if status.started():
-            return  # noop - make the user wait before starting a new one
-    else:
-        if status.not_started() or status.started():
-            return  # noop - one's already on the way
+    if manual and status.missing() and download_data.task:
+        download_data.task.revoke()
+    if status.not_started() or status.started():
```

Comment:
Ha, from the names of these methods it seems like this will always evaluate to true.

Not changed here, so out of scope: `not_started` is a terrible name. Should be changed to `pending`.

Reply:
I totally agree: in general our celery statuses are very confusing. `not_started` actually just returns: 
```py
task.state == 'PENDING'
```